[INFO] [2023-08-05 23:32:35,114] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:32:35,434] Updates were skipped successfully.

[INFO] [2023-08-05 23:32:35,438] Start polling.

[INFO] [2023-08-05 23:32:49,562] Stop polling...

[WARNING] [2023-08-05 23:32:49,565] Goodbye!

[INFO] [2023-08-05 23:33:23,438] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:33:23,755] Updates were skipped successfully.

[INFO] [2023-08-05 23:33:23,761] Start polling.

[INFO] [2023-08-05 23:37:18,631] Stop polling...

[WARNING] [2023-08-05 23:37:18,633] Goodbye!

[INFO] [2023-08-05 23:37:26,675] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:37:27,074] Updates were skipped successfully.

[INFO] [2023-08-05 23:37:27,088] Start polling.

[ERROR] [2023-08-05 23:37:29,350] Task exception was never retrieved
future: <Task finished name='Task-9' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=NameError("name 'is_admin' is not defined")>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 82, in start_handler
    if is_admin:
       ^^^^^^^^
NameError: name 'is_admin' is not defined
[ERROR] [2023-08-05 23:37:33,693] Task exception was never retrieved
future: <Task finished name='Task-18' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=NameError("name 'scanning' is not defined")>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 127, in start_scanning_sqlinj_handler
    results = scanning(generate_useragent(), msg.text)
              ^^^^^^^^
NameError: name 'scanning' is not defined
[INFO] [2023-08-05 23:37:57,355] Stop polling...

[WARNING] [2023-08-05 23:37:57,362] Goodbye!

[INFO] [2023-08-05 23:38:09,367] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:38:09,672] Updates were skipped successfully.

[INFO] [2023-08-05 23:38:09,678] Start polling.

[ERROR] [2023-08-05 23:38:14,671] Task exception was never retrieved
future: <Task finished name='Task-14' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=CantParseEntities("Can't parse entities: unclosed start tag at byte offset 0")>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 131, in start_scanning_sqlinj_handler
    await bot.send_message(msg.from_user.id, result, parse_mode='HTML')
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/bot.py", line 346, in send_message
    result = await self.request(api.Methods.SEND_MESSAGE, payload)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/base.py", line 236, in request
    return await api.make_request(await self.get_session(), self.server, self.__token, method, data, files,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/api.py", line 140, in make_request
    return check_result(method, response.content_type, response.status, await response.text())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/api.py", line 115, in check_result
    exceptions.BadRequest.detect(description)
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/utils/exceptions.py", line 140, in detect
    raise err(cls.text or description)
aiogram.utils.exceptions.CantParseEntities: Can't parse entities: unclosed start tag at byte offset 0
[INFO] [2023-08-05 23:38:35,558] Stop polling...

[WARNING] [2023-08-05 23:38:35,560] Goodbye!

[INFO] [2023-08-05 23:38:51,141] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:38:51,435] Updates were skipped successfully.

[INFO] [2023-08-05 23:38:51,439] Start polling.

[INFO] [2023-08-05 23:39:55,127] Stop polling...

[WARNING] [2023-08-05 23:39:55,129] Goodbye!

[INFO] [2023-08-05 23:43:03,713] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:43:04,020] Updates were skipped successfully.

[INFO] [2023-08-05 23:43:04,022] Start polling.

[ERROR] [2023-08-05 23:43:06,243] Task exception was never retrieved
future: <Task finished name='Task-9' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=NameError("name 'is_admin' is not defined")>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 88, in start_handler
    if is_admin:
       ^^^^^^^^
NameError: name 'is_admin' is not defined
[ERROR] [2023-08-05 23:43:08,465] Task exception was never retrieved
future: <Task finished name='Task-14' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=AttributeError("type object 'GetInfoAboutIP' has no attribute 'url'")>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 95, in get_info_ip_handler
    await GetInfoAboutIP.url.set()
          ^^^^^^^^^^^^^^^^^^
AttributeError: type object 'GetInfoAboutIP' has no attribute 'url'
[INFO] [2023-08-05 23:43:26,603] Stop polling...

[WARNING] [2023-08-05 23:43:26,605] Goodbye!

[INFO] [2023-08-05 23:43:35,114] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:43:35,431] Updates were skipped successfully.

[INFO] [2023-08-05 23:43:35,436] Start polling.

[ERROR] [2023-08-05 23:44:11,375] Task exception was never retrieved
future: <Task finished name='Task-15' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=MissingSchema("Invalid URL '35.173.69.207': No scheme supplied. Perhaps you meant https://35.173.69.207?")>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 106, in start_scanning_xss_handler
    result = scan_xss(msg.text, generate_useragent())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/modules/scanners/xss.py", line 103, in scan_xss
    forms = get_all_forms(url, fua)
            ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/modules/scanners/xss.py", line 36, in get_all_forms
    soup = bs(requests.get(url, headers=headers).content, "html.parser")
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/requests/sessions.py", line 575, in request
    prep = self.prepare_request(req)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/requests/sessions.py", line 486, in prepare_request
    p.prepare(
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/requests/models.py", line 368, in prepare
    self.prepare_url(url, params)
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/requests/models.py", line 439, in prepare_url
    raise MissingSchema(
requests.exceptions.MissingSchema: Invalid URL '35.173.69.207': No scheme supplied. Perhaps you meant https://35.173.69.207?
[INFO] [2023-08-05 23:44:42,162] Stop polling...

[WARNING] [2023-08-05 23:44:42,164] Goodbye!

[INFO] [2023-08-05 23:44:50,545] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:44:50,910] Updates were skipped successfully.

[INFO] [2023-08-05 23:44:50,913] Start polling.

[INFO] [2023-08-05 23:45:32,799] Stop polling...

[WARNING] [2023-08-05 23:45:32,802] Goodbye!

[INFO] [2023-08-05 23:46:00,198] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:46:00,502] Updates were skipped successfully.

[INFO] [2023-08-05 23:46:00,504] Start polling.

[INFO] [2023-08-05 23:46:41,933] Stop polling...

[WARNING] [2023-08-05 23:46:41,935] Goodbye!

[INFO] [2023-08-05 23:51:44,843] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:51:45,194] Updates were skipped successfully.

[INFO] [2023-08-05 23:51:45,196] Start polling.

[ERROR] [2023-08-05 23:52:13,011] Task exception was never retrieved
future: <Task finished name='Task-10' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=NameError("name 'is_admin' is not defined")>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 93, in start_handler
    if is_admin:
       ^^^^^^^^
NameError: name 'is_admin' is not defined
[ERROR] [2023-08-05 23:52:43,613] Task exception was never retrieved
future: <Task finished name='Task-20' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=CantParseEntities('Can\'t parse entities: unsupported start tag "img" at byte offset 1437')>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 113, in start_get_info_phone_handler
    await bot.send_message(msg.from_user.id, f'{result}', parse_mode='html')
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/bot.py", line 346, in send_message
    result = await self.request(api.Methods.SEND_MESSAGE, payload)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/base.py", line 236, in request
    return await api.make_request(await self.get_session(), self.server, self.__token, method, data, files,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/api.py", line 140, in make_request
    return check_result(method, response.content_type, response.status, await response.text())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/api.py", line 115, in check_result
    exceptions.BadRequest.detect(description)
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/utils/exceptions.py", line 140, in detect
    raise err(cls.text or description)
aiogram.utils.exceptions.CantParseEntities: Can't parse entities: unsupported start tag "img" at byte offset 1437
[INFO] [2023-08-05 23:53:51,754] Stop polling...

[WARNING] [2023-08-05 23:53:51,756] Goodbye!

[INFO] [2023-08-05 23:53:58,046] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:53:58,397] Updates were skipped successfully.

[INFO] [2023-08-05 23:53:58,402] Start polling.

[ERROR] [2023-08-05 23:54:07,200] Task exception was never retrieved
future: <Task finished name='Task-18' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=JSONDecodeError('Expecting value: line 1 column 1 (char 0)')>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/requests/models.py", line 971, in json
    return complexjson.loads(self.text, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 115, in start_get_info_phone_handler
    result = get_info_phonenumber(msg.text, generate_useragent())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/modules/osint/phone.py", line 35, in get_info_phonenumber
    info_data = requests.get(url, headers=headers).json()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/requests/models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[INFO] [2023-08-05 23:54:51,598] Stop polling...

[WARNING] [2023-08-05 23:54:51,600] Goodbye!

[INFO] [2023-08-05 23:55:00,390] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:55:00,717] Updates were skipped successfully.

[INFO] [2023-08-05 23:55:00,719] Start polling.

[INFO] [2023-08-05 23:55:38,226] Stop polling...

[WARNING] [2023-08-05 23:55:38,228] Goodbye!

[INFO] [2023-08-05 23:56:38,723] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:56:39,065] Updates were skipped successfully.

[INFO] [2023-08-05 23:56:39,071] Start polling.

[ERROR] [2023-08-05 23:56:50,194] Task exception was never retrieved
future: <Task finished name='Task-14' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=CantParseEntities('Can\'t parse entities: unsupported start tag "img" at byte offset 1437')>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 117, in start_get_info_phone_handler
    await bot.send_message(msg.from_user.id, f'{result}', parse_mode='html')
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/bot.py", line 346, in send_message
    result = await self.request(api.Methods.SEND_MESSAGE, payload)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/base.py", line 236, in request
    return await api.make_request(await self.get_session(), self.server, self.__token, method, data, files,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/api.py", line 140, in make_request
    return check_result(method, response.content_type, response.status, await response.text())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/bot/api.py", line 115, in check_result
    exceptions.BadRequest.detect(description)
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/utils/exceptions.py", line 140, in detect
    raise err(cls.text or description)
aiogram.utils.exceptions.CantParseEntities: Can't parse entities: unsupported start tag "img" at byte offset 1437
[INFO] [2023-08-05 23:57:30,017] Stop polling...

[WARNING] [2023-08-05 23:57:30,020] Goodbye!

[INFO] [2023-08-05 23:57:40,330] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:57:40,613] Updates were skipped successfully.

[INFO] [2023-08-05 23:57:40,616] Start polling.

[ERROR] [2023-08-05 23:58:12,939] Task exception was never retrieved
future: <Task finished name='Task-24' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=KeyError('region')>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 115, in start_get_info_phone_handler
    result = get_info_phonenumber(msg.text, generate_useragent())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/modules/osint/phone.py", line 42, in get_info_phonenumber
    Регион: {info_data["region"]["name"]}
             ~~~~~~~~~^^^^^^^^^^
KeyError: 'region'
[INFO] [2023-08-05 23:58:56,275] Stop polling...

[WARNING] [2023-08-05 23:58:56,278] Goodbye!

[INFO] [2023-08-05 23:59:05,142] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:59:05,560] Updates were skipped successfully.

[INFO] [2023-08-05 23:59:05,566] Start polling.

[INFO] [2023-08-05 23:59:15,281] Stop polling...

[WARNING] [2023-08-05 23:59:15,284] Goodbye!

[INFO] [2023-08-05 23:59:25,098] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-05 23:59:25,418] Updates were skipped successfully.

[INFO] [2023-08-05 23:59:25,423] Start polling.

[ERROR] [2023-08-06 00:05:42,606] Task exception was never retrieved
future: <Task finished name='Task-86' coro=<Dispatcher._process_polling_updates() done, defined at /home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py:407> exception=AttributeError("'NoneType' object has no attribute 'get'")>
Traceback (most recent call last):
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/venv/lib/python3.11/site-packages/aiogram/dispatcher/handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/punisher.py", line 182, in start_scanning_sqlinj_handler
    results = scanning(generate_useragent(), msg.text)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/modules/scanners/sql_injection.py", line 161, in scanning
    content = scan_sql_injection(session, url)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/modules/scanners/sql_injection.py", line 123, in scan_sql_injection
    form_details = get_form_details(form)
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/okulus/Tools/Punisher-Tool/modules/scanners/sql_injection.py", line 57, in get_form_details
    method = form.attrsession.get("method", "get").lower()
             ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
[INFO] [2023-08-06 00:07:20,318] Stop polling...

[WARNING] [2023-08-06 00:07:20,321] Goodbye!

[INFO] [2023-08-06 00:07:46,375] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-06 00:07:46,661] Updates were skipped successfully.

[INFO] [2023-08-06 00:07:46,662] Start polling.

[INFO] [2023-08-06 00:12:15,867] Stop polling...

[WARNING] [2023-08-06 00:12:15,869] Goodbye!

[INFO] [2023-08-06 00:12:23,492] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-06 00:12:23,772] Updates were skipped successfully.

[INFO] [2023-08-06 00:12:23,778] Start polling.

[INFO] [2023-08-06 00:13:19,809] Stop polling...

[WARNING] [2023-08-06 00:13:19,811] Goodbye!

[INFO] [2023-08-06 00:13:28,141] Bot: Punisher Tool⚡️ [@PunisherToolKit_bot]

[WARNING] [2023-08-06 00:13:28,547] Updates were skipped successfully.

[INFO] [2023-08-06 00:13:28,549] Start polling.

[INFO] [2023-08-06 00:15:05,948] Stop polling...

[WARNING] [2023-08-06 00:15:05,950] Goodbye!

[WARNING] [2023-08-06 00:15:28,255] Goodbye!

